platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/tipset/TipsetEventCreationManagerFactory.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.event.tipset;
+
+import static com.swirlds.common.threading.interrupt.Uninterruptable.abortAndThrowIfInterrupted;
+
+import com.swirlds.base.time.Time;
+import com.swirlds.common.context.PlatformContext;
+import com.swirlds.common.stream.Signer;
+import com.swirlds.common.system.NodeId;
+import com.swirlds.common.system.SoftwareVersion;
+import com.swirlds.common.system.address.AddressBook;
+import com.swirlds.common.system.platformstatus.PlatformStatus;
+import com.swirlds.common.threading.framework.QueueThread;
+import com.swirlds.common.threading.manager.ThreadManager;
+import com.swirlds.platform.StartUpEventFrozenManager;
+import com.swirlds.platform.event.EventIntakeTask;
+import com.swirlds.platform.event.GossipEvent;
+import com.swirlds.platform.eventhandling.EventTransactionPool;
+import com.swirlds.platform.observers.ConsensusRoundObserver;
+import com.swirlds.platform.observers.EventObserverDispatcher;
+import com.swirlds.platform.observers.PreConsensusEventObserver;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Objects;
+import java.util.Random;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+/**
+ * A factory for creating {@link TipsetEventCreator} instances.
```

Comment:
This factory is creating instances of `TipsetEventCreationManager`, not `TipsetEventCreator`

Reply:
fixed